sycl/test-e2e/KernelCompiler/kernel_compiler.cpp

```diff
@@ -8,6 +8,9 @@
 
 // REQUIRES: ocloc
 
+// UNSUPPORTED: (gpu-intel-dg2 && level_zero) || (gpu-intel-pvc && level_zero)
+// seems to be an incompatibility with the KernelCompiler on L0 with DG2 and PVC
```

Comment:
```suggestion
// Seems to be an incompatibility with the KernelCompiler on L0 with DG2 and PVC.
```

Is this reported internally?

Reply:
Yes, opening a ticket on it this morning, after I test one more thing